.github/workflows/reusable-docker-build-publish.yml

```
@@ -288,6 +293,46 @@ jobs:
             exit 1
           fi
 
+      - name: Get repo trunk branch
+        id: get-trunk
+        if: ${{ inputs.docker-cache-behaviour == 'write-on-trunk' }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_WORKFLOW_REPOSITORY: ${{ inputs.github-workflow-repository }}
+        run: |
+          trunk=$(gh repo view ${GITHUB_WORKFLOW_REPOSITORY} --json defaultBranchRef --jq .defaultBranchRef.name)
```

```suggestion
          trunk=$(gh repo view "${GITHUB_WORKFLOW_REPOSITORY}" --json defaultBranchRef --jq .defaultBranchRef.name)
```

To appease shellcheck